dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -71,7 +77,10 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
             UnRegisterComServer(storedComActivatorString);
             UnRegisterAppIdentifierFromRegistry();
 
-            // TODO: Remove ToastGuid reference from LRP
+            wil::unique_cotaskmem_string processName;
+            THROW_IF_FAILED(GetCurrentProcessPath(processName));
+            auto notificationPlatform{ PushNotificationHelpers::GetNotificationPlatform() };
+            THROW_IF_FAILED(notificationPlatform->RemoveToastRegistration(processName.get()));
```

Comment:
We should also call this when the Push Activator is unregistered right?

Reply:
RemoveToastRegistration is called by UnregisterLongRunningActivator which is called when unregistering Push activator.